pkg/genericapiserver/genericapiserver.go

```diff
@@ -128,6 +131,20 @@ type APIGroupVersionOverride struct {
 	ResourceOverrides map[string]bool
 }
 
+// Info about an API group.
+type APIGroupInfo struct {
+	GroupMeta latest.GroupMeta
```

Comment:
Yes thats right.
I will try to come up with a better place for GroupMeta and move it in another PR.